pkgs/development/ocaml-modules/qcheck/ounit.nix

```
@@ -1,10 +1,17 @@
-{ buildDunePackage, qcheck-core, ounit }:
+{ lib, buildDunePackage, ocaml, qcheck-core, ounit }:
 
 buildDunePackage {
   pname = "qcheck-ounit";
 
   inherit (qcheck-core) version src;
 
+  duneVersion = "3";
```

```suggestion
  duneVersion = "3";
```
IMO, `duneVersion` is configurePhase. I'd expect it after patchPhase before buildPhase.